treescript/src/treescript/mercurial.py

```diff
@@ -335,7 +341,7 @@ def _count_outgoing(output):
 
 
 # push {{{1
-async def push(config, task, repo_path):
+async def push(config, task, repo_path, source_repo=None, revision=None):
```

Comment:
I can change it to target - it was called that because it gets the default value from `metadata.source`, and I felt source_repo was clearer, but I can see arguments both ways.